src/maker/config.rs

```diff
@@ -31,16 +31,13 @@ pub struct MakerConfig {
     pub socks_port: u16,
     /// Authentication password for Tor interface
     pub tor_auth_password: String,
-    /// DNS Tor address. Change this to connect to a different DNS server
-    pub dns_address: String,
     /// Minimum amount in satoshis that can be swapped
     pub min_swap_amount: u64,
     /// Fidelity Bond amount in satoshis
     pub fidelity_amount: u64,
     /// Fidelity Bond relative timelock in number of blocks
     pub fidelity_timelock: u32,
     /// Connection type (TOR or CLEARNET)
-    ///
     /// # Deprecated
```

Comment:
Can we try to remove the connection type from config too? Its deprecated and users are not allowed to change from Tor. 

Though this is used in the tests, so we can keep it in the maker APIs internally. Just removing it from the config file will be enough. 

Can be done in a followup PR.